src/DacpacTool/PackageAnalyzer.cs

```diff
@@ -21,6 +21,21 @@ public PackageAnalyzer(IConsole console, string rulesExpression)
             BuildRuleLists(rulesExpression);
         }
 
+        public void AddRulesFile(FileInfo inputFile)
+        {
+            // Make sure the file exists
+            if (!inputFile.Exists)
+            {
+                throw new ArgumentException($"Unable to find rules file {inputFile}", nameof(inputFile));
+            }
+
+            if (inputFile.Directory.Name.Equals("rules", StringComparison.OrdinalIgnoreCase)
```

Comment:
I don't generally like requiring files to be under a certain path, but I'm okay with it in this case, since we don't really have another mechanism. The `Include` elements in the `.csproj` file don't include anything like a `Type` attribute and the `csproj` format isn't within the scope of our project.

Reply:
Agree, but it was the best solution I could brew up